typescript/libs/view_helpers.py

```diff
@@ -36,7 +37,7 @@ def __init__(self, filename, cc):
 _file_map_on_worker = dict()
 
 
-def get_info(view):
+def get_info(view, open_if_not_found=True):
```

Comment:
This param name is confusing. What does it mean exactly? (How do you open something if you cant find it?)